modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -657,7 +657,13 @@ private ObjectDefinition _toObjectDefinition(
 				id = objectDefinition.getObjectDefinitionId();
 				label = LocalizedMapUtil.getLanguageIdMap(
 					objectDefinition.getLabelMap());
-				modifiable = objectDefinition.getModifiable();
+
+				if (!GetterUtil.getBoolean(
```

Comment:
I think this condition is wrong, it must validate if the feature flag is active/true